src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -145,4 +146,29 @@ public void parseCommand_priorityMissingLevel_throwsParseException() {
                                 PriorityCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased()));
     }
 
+    @Test
+    public void parseCommand_findTask() throws Exception {
+        Index index = INDEX_FIRST_PERSON;
+
+        FindTaskCommand expectedCommand = new FindTaskCommand(index);
+
+        FindTaskCommand command = (FindTaskCommand) parser.parseCommand(
+                FindTaskCommand.COMMAND_WORD + " " + index.getOneBased());
+        assertEquals(expectedCommand, command);
+    }
+
+    @Test
+    public void parseCommandFindTaskInvalidFormatThrowsParseException() {
+        assertThrows(ParseException.class,
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindTaskCommand.MESSAGE_USAGE), ()
+                        -> parser.parseCommand(FindTaskCommand.COMMAND_WORD + " abc"));
+    }
+
+    @Test
+    public void parseCommandFindTaskMissingIndexThrowsParseException() {
+        assertThrows(ParseException.class,
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindTaskCommand.MESSAGE_USAGE), ()
+                        -> parser.parseCommand(FindTaskCommand.COMMAND_WORD + " "));
+    }
+
```

Comment:
Can consider a test case that appends some string value after a valid command? i.e. findtask 1 test